main_test.go

```diff
@@ -3085,6 +3086,18 @@ func TestLogsEndpoints(t *testing.T) {
 				assert.Equal(t, "invalid date time format (RFC 3339 needed)", response["detail"])
 			},
 		},
+		{
+			description: `GET with "search" query param`,
+			query:       "GET /v1/logs?search=debug",
```

Comment:
Adapting to fixtures

```suggestion
			query:       "GET /v1/logs?search=bad publiccode.yml",
```